tools/check_release.py

```diff
@@ -432,17 +483,26 @@ def get_latest_library_versions(repo_path):
                         # Internal compiler error thus retry
                         continue
                     else:
-                        # Genuine compilation error, thus print it out
-                        logging.error("\t\tError: %s\n", mesg)
+                        # Actual error thus move on to next compilation
+                        failures.append([test, target])
+                        break
                                     
                 passes += (int)(result)
                 break
             else:
-                logging.error("\t\tProgram/Target compilation failed due to internal errors. Removing from considered list!\n")
+                logging.error("\t\tCompilation failed due to internal errors.\n")
+                logging.error("\t\tSkipping test/target combination!\n")
                 total -= 1   
+                skipped.append([test, target])
                 
+    logging.info(" SUMMARY OF COMPILATION RESULTS")                
+    logging.info(" ------------------------------\n")                
```

Comment:
Same comment as previous, as to human eye readability . but again I don't really care that much..